master/src/main/java/eu/unifiedviews/master/api/PipelineResource.java

```diff
@@ -120,12 +148,24 @@ public PipelineDTO clonePipeline(@PathParam("pipelineid") String id, PipelineDTO
             throw new ApiException(Response.Status.NOT_FOUND, String.format("ID=%s is not valid pipeline ID", id));
         }
         try {
+            // try to get pipeline
             pipeline = pipelineFacade.getPipeline(Long.parseLong(id));
             if (pipeline == null) {
                 throw new ApiException(Response.Status.NOT_FOUND, String.format("Pipeline with id=%s doesn't exist!", id));
             }
+            // try to get user
+            User user =  userFacade.getUserByExtId(pipelineDTO.getUserExternalId());
+            if(user == null) {
+                throw new ApiException(Response.Status.NOT_FOUND, String.format("User '%s' could not be found! Pipeline could not be created.", pipelineDTO.getUserExternalId()));
+            }
+            // try to get organization
+            Organization organization = userFacade.getOrganizationByName(pipelineDTO.getOrganizationExternalId());
+            if(organization == null) {
```

Comment:
The same as for createPipeline, see above. If I copy pipeline using UV GUI, and config.properties has param "ownership.type" is set to "user", then, the copied pipeline has NULL organization, which is correct.